test/testvm.py

```diff
@@ -611,6 +611,9 @@ def _start_qemu(self, maintain=False, tty=False, monitor=None):
         if tty:
             return subprocess.Popen(cmd)
 
+        quoted = " ".join("'%s'" % x for x in cmd)
+        cmd = [ "/bin/sh", "-x", "-c", quoted + " | tee /tmp/qemup.log" ]
```

Comment:
Will put it in the run directory, and give it the process name. The idea is not to guarantee that it won't be overwritten even, but to provide enough for forensics.